src/database/DataTypes/index.js

```diff
@@ -8,6 +8,7 @@ import Realm from 'realm';
 export class Address extends Realm.Object {}
 export class ItemCategory extends Realm.Object {}
 export class ItemDepartment extends Realm.Object {}
+export class Report extends Realm.Object {}
```

Comment:
Slightly different this time round, I think, because we would want to use `get` and `set` it would be good to have it's own file.

I think adding a file to `dataTypes/Report.js` and adding that line there. It's a small change, but it will make it easier to add getters and setters later.

Reply:
Yes, thanks! I was just reading #1709 (related with this) and thinking about the same. I will create that file to later add get() and set() there